docs/manifest.md

```diff
@@ -249,6 +267,25 @@ consent with granting access to Outreach API in his name. These URLs can be the
 separate URL, but in both cases, it has to be implemented in a way matching
 [Outreach API access requirements](outreach-api.md).
 
+## Outreach S2S API access section ("apiS2S")
```

Comment:
Do we want to document this file in Dev Portal https://developers.outreach.io ?

Reply:
eventually, once the feature is release we should transform this document https://docs.google.com/document/d/10SPaCnvsJsexOMnQ7MSzMtWE_YLqRhk9iQaHiGyPUnw/edit to the portal

Reply:
Yes, for every new feature in developer ecosystem we should implicitly calculate with a doc article on developers.outreach.io.